src/frontend/src/lib/styles/global/button.scss

```diff
@@ -177,4 +177,11 @@ button {
 		opacity: 1;
 		--box-shadow-opacity: 0.75;
 	}
+
+	&[disabled],
+	&.disabled {
+		background: var(--color-grey);
+		color: var(--color-white);
+		opacity: 1;
```

Comment:
My mistake! It should have been put inside the `primary` and `secondary` specifics CSS props. Changed!

(Those classes never have a specific `opacity` setting set during the rest of the code)

Reply:
What I mean is that we have such button as following:

```
<button class="primary mt-6" disabled={$isBusy} class:opacity-50={$isBusy} on:click={openReceive}>
		{replacePlaceholders($i18n.info.ethereum.how_to, {
			$ckToken: ckTokenSymbol
		})}</button
	>
```

Setting `opacity: 1` in the global style won't clash with `class="primary" class:opacity-50={$isBusy}`?

Reply:
Thanks for the feedback

Reply:
As discussed with Artem, this will be the default for all `primary` and `secondary` buttons, so I am removing the opacity class where needed with PR #2365